Sources/_StringProcessing/ConsumerInterface.swift

```diff
@@ -109,9 +107,20 @@ extension AST.Atom {
         // TODO: alias? casing?
         $0.name == name || $0.nameAlias == name
       }
+      
+    case .any:
+      return { input, bounds in
+        let curIndex = bounds.lowerBound
+        if !opts.contains(.singleLine) && input[curIndex].isNewline {
```

Comment:
Again, `.singleLine` is one particular spelling. It might be that a 1-1 happens to fall out, but it's a bit of a smell. We could have a `var isSingleLine: Bool` or a `var lineMode: LineMode` enum on the options type.